tests/ZeroC.Slice.Tests/CustomTypeTests.cs

```diff
@@ -9,32 +10,207 @@ public class CustomTypeTests
     [Test]
     public void Decode_custom_type()
     {
+        // Arrange
         var buffer = new MemoryBufferWriter(new byte[256]);
         var encoder = new SliceEncoder(buffer, SliceEncoding.Slice2);
         var expected = new MyCustomType { Flag = true, Value = 10 };
         encoder.EncodeCustomType(expected);
         var decoder = new SliceDecoder(buffer.WrittenMemory, SliceEncoding.Slice2);
 
+        // Act
         var value = new StructWithCustomTypeField(ref decoder);
 
+        // Assert
         Assert.That(value.M, Is.EqualTo(expected));
         Assert.That(decoder.Consumed, Is.EqualTo(buffer.WrittenMemory.Length));
     }
 
     [Test]
     public void Encode_custom_type()
     {
+        // Arrange
         var buffer = new MemoryBufferWriter(new byte[256]);
         var encoder = new SliceEncoder(buffer, SliceEncoding.Slice2);
         var expected = new StructWithCustomTypeField(new MyCustomType { Flag = true, Value = 10 });
 
+        // Act
         expected.Encode(ref encoder);
 
+        // Assert
         var decoder = new SliceDecoder(buffer.WrittenMemory, SliceEncoding.Slice2);
         var value = decoder.DecodeCustomType();
         Assert.That(expected.M, Is.EqualTo(value));
         Assert.That(decoder.Consumed, Is.EqualTo(buffer.WrittenMemory.Length));
     }
+
+    [Test]
+    public void Decode_sequence_of_custom_types()
```

Comment:
If your test is for some reason Slice1-only or Slice2-only, it needs Slice1/resp. Slice2 in its name.

Reply:
Should I add this even for the test that's "Decode_Class..."
I looked at the class tests and we don't encoding-prefix them, but maybe that's not a good role model.